### ISSUE TITLE
Possibility to save logs

### ISSUE THREAD

**Issue by TheGodlyLuzer** [altmp/altv-issues#1484]
### Description of the problem

When starting a Server  all logs are overwritten and therefore lost. When using an automated serverrestart there is no chance to take a look at the logs, unless you save them yourself. You could save it yourself, but it would be great to have a possibility to automatically save logs. Would save a lot of headaches.

### Desired solution for the problem

Add a bool to altv.cfg. When set to true, all logs are save to a "logs" folder.

**Comment by deluvas1911** [comment]
You can use the [command line arg](http://docs.altv.mp/articles/commandlineargs.html) `--logfile [path]` to create per session log files in your automated restart script

If the other option by setting e.g. a value in the cfg file will be implemented is something a dev has to decide

**Comment by TheGodlyLuzer** [comment]
Yeah there are lots of possibilites to solve it, just would be simpler to have a direct solution 😄
